enterprise/web.el/src/org/netbeans/modules/web/el/ELTypeUtilities.java

```diff
@@ -20,17 +20,7 @@
 
 import com.sun.el.parser.*;
 import java.net.URL;
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Deque;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
This is a problem. It breaks the jdeps(1) dependency mapping. In addition, it kills the modularity of the project.

For this reason, I have to give it a -1. Can you please fix changes like this and resubmit.

I'm going to stop here, but there are several others that would receive the same comment.

Reply:
@BradWalker ty